### PR TITLE
Remove missing functions from interfaces, add missing script import

### PR DESCRIPTION
**Pull request by TwitchBronBron**
**Changes**

Just fixing a few small issues in the codebase.
1. `SeriesData.xml`, `TvEpisodeData.xml`, and `TVSeasonData.xml` were exposing a function in the interface that isn't defined.
2. When `Items.brs` is referenced from within `components/ItemGrid2/LoadItemsTask2.xml`, the call to `getDeviceProfile()` would fail because `LoadItemsTask2.xml` doesn't reference `pkg:/source/utils/deviceCapabilities.brs`

image
 image

### CODE REVIEW

File: components/data/SeriesData.xml [hunk 1]
```diff
@@ -5,7 +5,6 @@
     <field id="seasons" type="associativearray" />
     <field id="nextup" type="associativearray" />
     <field id="image" type="node" onChange="setPoster" />
-    <function name="loadSeasons" />
```
Comment (TwitchBronBron) [inline components/data/SeriesData.xml:8]: I did a global search through the codebase, and `loadSeasons` isn't defined anywhere.

File: components/data/TVSeasonData.xml [hunk 2]
```diff
@@ -7,7 +7,6 @@
     <field id="posterURL" type="string" />
     <field id="overview" type="string" />
     <field id="json" type="associativearray" onChange="setFields" />
-    <function name="getPoster" />
```
Comment (TwitchBronBron) [inline components/data/TVSeasonData.xml]: I did a global search through the codebase, and `getPoster` isn't defined anywhere.

**Comment by neilsb** [comment]
That looks great. Thanks for this. I'll aim to look at it in detail and get it tested and merged tomorrow.

**Review by neilsb — APPROVED** [review]

**Comment by neilsb** [comment]
We could go with getting basic error checking done automatically as part of a Automated Build Pipeline.

**Comment by TwitchBronBron** [comment]
I'd be more than happy to set that up and submit a PR, if you want? I'll open a separate GH issue to track this and we can work out the details there.

**Pull request merged**